packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/DeviceOrientationManager.java

```diff
@@ -236,6 +237,8 @@ static void handleOrientationChange(
    *
    * @return The current user interface orientation.
    */
+  // Configuration.ORIENTATION_SQUARE is deprecated.
```

Comment:
Since this no longer is supported or used, could you go ahead and delete that case?

Reply:
Oh gotcha!